lib/server/src/server.rs

```diff
@@ -859,4 +879,38 @@ impl Registry for TrowServer {
         };
         Ok(Response::new(reply))
     }
-}
+
+    async fn get_metrics(
```

Comment:
I'd say move this into `metrics.rs`, but I think that won't work as this is inside the `impl`.

Reply:
Refactored to move some of the logic away.